src/components_1/MainImage.js

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
 
-const MainImage = () => {
+import purdue from '../assets/purdue.jpg';
+
+const MainImage = () => (
     <>
+        <img src = {purdue}/>
```

Comment:
I am not sure why styling an image isn't working. I will take a look at 11:00 but I have an alternate solution for you